app/scripts/templates/settings/avatar.mustache

```diff
@@ -3,10 +3,15 @@
     <header class="settings-unit-summary">
       <h2 class="settings-unit-title">{{#t}}Account picture{{/t}}</h2>
     </header>
-    <button class="settings-button secondary settings-unit-toggle"
-      data-href="settings/avatar/change">
-      {{#avatar}}<span class="change-button">{{#t}}Change…{{/t}}</span>{{/avatar}}
-      {{^avatar}}<span class="add-button">{{#t}}Add…{{/t}}</span>{{/avatar}}
-    </button>
+    {{#avatar}}
+      <button class="settings-button secondary settings-unit-toggle hidden" data-href="settings/avatar/change">
```

Comment:
The `hidden` class doesn't seem to do anything here, I'm not sure what the original intent was. Can you remove it?